egs/librimix/ConvTasNet/local/get_text.py

```diff
@@ -0,0 +1,28 @@
+import argparse
+import glob
+import os
+import pandas as pd
+
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--libridir", required=True, type=str)
+parser.add_argument("--outfile", required=True, type=str)
+parser.add_argument("--split", type=str, default="train-360")
+
+args = parser.parse_args()
+
+libridir = f"LibriSpeech/{args.split}"
```

Comment:
`libridir = os.path.join(args.libridir, args.split)`

Reply:
Feel free to push to this branch. I won't touch it for a small while